pyproject.toml

```diff
@@ -95,13 +95,90 @@ dev = [
     "types-psutil==5.8.23",
     "types_requests==2.31.0.20240406",
     "types-protobuf==5.26.0.20240422",
+    "deptry>=0.14"
```

Comment:
please pin dev deps to a specific version so we can ensure all our developers have exactly the same tools.